### PR TITLE
VAGOV-6263: Fixing ie11 image display issue.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
Updates two classes to have max-height treatment (only)

## Testing done
Visual

## Screenshot
@cvalarida - Left side is ie11, right is firefox. Note how the left hand side is stretched / distorted as object-fit property is not recognized by ie11. 
image

### CODE REVIEW

File: src/site/teasers/bio.drupal.liquid [hunk 1]
```diff
@@ -42,11 +42,12 @@ Example data:
...
-            {% assign image = node.fieldMedia.entity.image %}
-            <img class="bio-image" src="{{ image.derivative.url }}" alt="{{ image.alt }}"
-                title="{{ image.title }}">
-        </div>
+    <div class="vads-u-margin-right--2 vads-u-flex--auto">
+        {% assign image = node.fieldMedia.entity.image %}
+        <img data-object-fit="cover" class="bio-image"
+            src="{{ image.derivative.url }}" alt="{{ image.alt }}"
+            title="{{ image.title }}">
+    </div>
```
Comment (swirtSJW) [inline src/site/teasers/bio.drupal.liquid]: CS: Lost its indentation to the 'if' logic.

File: src/site/teasers/bio.drupal.liquid [hunk 2]
```diff
@@ -90,14 +91,14 @@ Example data:
         </p>
         {% endif %}
         {% if node.fieldEmailAddress %}
-            <p class="
```
Comment (swirtSJW) [inline src/site/teasers/bio.drupal.liquid]: CS: lost indentaition relative to control structure.

File: src/site/teasers/bio.drupal.liquid [hunk 3]
```diff
@@ -90,14 +91,14 @@ Example data:
...
         vads-u-margin--0
         vads-u-margin-bottom--0
         vads-u-font-size--base
         medium-screen:vads-u-font-size--lg">
-               <b>Email:</b> <a target="blank"
-                          href="{{ node.fieldEmailAddress }}">{{ node.fieldEmailAddress }}</a>
-            </p>
+            <b>Email:</b> <a target="blank"
+                href="{{ node.fieldEmailAddress }}">{{ node.fieldEmailAddress }}</a>
+        </p>
```
Comment (swirtSJW) [inline src/site/teasers/bio.drupal.liquid]: CS: lost indentation relative to control structure.

**Review by swirtSJW — CHANGES_REQUESTED** [review]
Logic all looks good.  Some odd loss of code style indentation relative to control structure  that should likely be fixed.

File: src/site/teasers/bio.drupal.liquid (hunk 1 above)
Comment (ethanteague) [reply src/site/teasers/bio.drupal.liquid]: Yeah, I'm not really sure what our standard for liquid is, @swirtSJW - we have a mixture of styles in different liquid files. My vscode liquid formatter wants to bring code blocks in conditionals flush with the conditional. Do you have a vscode tool in place that is alerting on this / formatting differently?

**Review by cvalarida — COMMENTED** [review]
Could you provide screenshots of the problem and how this fixes it?

Is there a way that this problem can be solved without using `data-object-fit` so we don't need to ship this extra code on all pages for all browsers?

**Comment by ethanteague** [comment]
> Could you provide screenshots of the problem and how this fixes it?
> 
> Is there a way that this problem can be solved without using `data-object-fit` so we don't need to ship this extra code on all pages for all browsers?

Just uploaded a screenshot and revised description with more info, @cvalarida. I googled this issue with ie11 and object-fit, and I don't think there is another way to skin this without using a polyfill type method.

**Comment by cvalarida** [comment]
I guess I'm confused why `object-fit` is needed in the first place. It looks like the problem is with cropping the image.

On way we can bypass this issue is by:
- Removing the `height` and `data-fit` from the `.bio-image`
- Adding `max-height` and `overflow: hidden` to the parent `<div>`

At any rate, I don't think it makes sense to increase the size of the bundle for all pages in all browsers when it's only needed for a very, very specific case.

**Comment by ethanteague** [comment]
> I guess I'm confused why `object-fit` is needed in the first place. It looks like the problem is with cropping the image.
> 
> On way we can bypass this issue is by:
> 
>     * Removing the `height` and `data-fit` from the `.bio-image`
> 
>     * Adding `max-height` and `overflow: hidden` to the parent `<div>`
> 
> 
> At any rate, I don't think it makes sense to increase the size of the bundle for all pages in all browsers when it's only needed for a very, very specific case.

I hear you, but I think this is a big deal to Pittsburgh - we tried methods like you suggested before, with mixed results. That said, I really don't have enough background to speak more intelligently about why we have to do this, so I'll defer to @jefflbrauer & @kevwalsh

**Comment by cvalarida** [comment]
My understanding of the problem:
- The images need to be exactly `176px` tall and `144px` wide
- The images can't be squashed

Is there more that I'm missing?

File: src/site/teasers/bio.drupal.liquid (hunk 1 above)
Comment (swirtSJW) [reply src/site/teasers/bio.drupal.liquid]: I don't, my tired eyes just like the readability of nested code.

I could not find official Code style definitions but I see the example provided here 
https://shopify.github.io/liquid/basics/operators/  use what I would considered  standard.

**Review by cvalarida — APPROVED** [review]
Looks great! It's probably worth updating the PR description for posterity since this no longer includes the polyfill.

**Review by swirtSJW — APPROVED** [review]
LGTM

**Pull request merged**